test/debug_naming_tests.cc

```diff
@@ -0,0 +1,65 @@
+#include "task.h"
+#include "task_manager.h"
+#include "types.h"
+
+#include <catch2/catch_template_test_macros.hpp>
+#include <catch2/catch_test_macros.hpp>
+#include <catch2/matchers/catch_matchers_string.hpp>
+
+#include <celerity.h>
+
+#include "test_utils.h"
+
+namespace celerity {
+namespace detail {
```

Comment:
We used to put tests into `celerity::detail` in the beginning, but new tests should be in the global namespace. I would suggest adding
```c++
using namespace celerity;
using namespace celerity::detail;
```
here instead.

Reply:
Good to know :+1: